RecoLocalCalo/HcalRecProducers/plugins/alpaka/HcalSiPMCharacteristicsESProducer.cc

```diff
@@ -0,0 +1,65 @@
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "CondFormats/HcalObjects/interface/alpaka/HcalSiPMCharacteristicsDevice.h"
+#include "CondFormats/HcalObjects/interface/HcalSiPMCharacteristicsSoA.h"
+#include "CondFormats/DataRecord/interface/HcalSiPMCharacteristicsRcd.h"
+
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ESGetToken.h"
```

Comment:
Doesn't seem to be needed
```suggestion
```
but then you would probably need
```cpp
#include "FWCore/Utilities/interface/ESGetToken.h"
```

Reply:
removed